### PR TITLE
[LLM] add Qwen-7B-Chat to PaddleNLP unit test

### PR DESCRIPTION
**Pull request by ziangqin-baidu**
### PR types

add Qwen-7B-Chat in PaddleNLP unit-test (test_predictor)

### PR changes

Models

### Description

bug fix: 
  1. fix default `padding_side` from right to left in batch inference of Qwen
  2. remove logits post-process in GenerationMixin > greedy_search
unit test: add Qwen into unit test

### CODE REVIEW

**Review by wj-Mcat — APPROVED** [review]
LGTM
